### PR TITLE
Change "Last BP" to "Last Seen" in Search Results

### PR DESCRIPTION
**Pull request by msasikanth**
https://www.pivotaltracker.com/story/show/170068604

### CODE REVIEW

File: app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt [hunk 1]
```diff
@@ -2251,6 +2251,118 @@ class PatientRepositoryAndroidTest {
...
+    val fiveSecondsLater = now.plusSeconds(5)
+
+    val uuidOfPatientA = UUID.fromString("e2a49529-3b46-4c9e-a7bc-af3090f1ecbb")
+    val uuidOfBp1OfPatientA = UUID.fromString("588b0d8c-6dce-4664-863f-8c79c06c6c3c")
+    val uuidOfBp2OfPatientA = UUID.fromString("ba875b98-4ca0-4097-bbc7-56041b377cf2")
+    val uuidOfBp3OfPatientA = UUID.fromString("4b92601e-4a86-40de-b6b7-707f9122a81b")
+    createPatient(patientUuid = uuidOfPatientA, patientName = "Patient with latest BP in Facility A")
+    recordBloodSugar(uuid = uuidOfBp1OfPatientA, patientUuid = uuidOfPatientA, facilityUuid = facilityA.uuid, recordedAt = oneSecondLater)
+    recordBp(uuid = uuidOfBp2OfPatientA, patientUuid = uuidOfPatientA, facilityUuid = facilityB.uuid, recordedAt = oneSecondEarlier)
+    recordBloodSugar(uuid = uuidOfBp3OfPatientA, patientUuid = uuidOfPatientA, facilityUuid = facilityA.uuid, recordedAt = now)
```
Comment (vinaysshenoy) [inline app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt]: Some of these are blood sugars and the others are BPs. Can we rename the variables to reflect that?

File: app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt [hunk 2]
```diff
@@ -2251,6 +2251,118 @@ class PatientRepositoryAndroidTest {
         .inOrder()
   }
 
+  @Test
+  fun querying_all_patients_in_a_facility_should_set_the_the_latest_recorded_measurement_recorded_on_within_any_facility() {
```
Comment (vinaysshenoy) [inline app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt]: Can we also add another test that a patient shows up in the list of "All patients in facility" if they have only blood sugar measurements in a specific facility?

Comment (msasikanth) [reply app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt]: I have added a test for All patients in the facility in the previous PR. Should I still add it?
`querying_all_patients_in_a_facility_should_return_only_patients_in_that_facility_ordered_by_name`

Comment (vinaysshenoy) [reply app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt]: Oh wait, I didn't check the entire file. Will check now.

**Pull request merged**